lib/models/addon-discovery.js

```diff
@@ -21,11 +20,10 @@ const heimdall = require('heimdalljs');
   @extends CoreObject
   @constructor
 */
-let AddonDiscovery = CoreObject.extend({
-  init(ui) {
-    this._super.apply(this, arguments);
+class AddonDiscovery {
```

Comment:
Don't we need to extend `CoreObject` here? Or are we calling this internal-only?